train.py

```diff
@@ -0,0 +1,27 @@
+import hydra
+from omegaconf import DictConfig, OmegaConf
+
+from pytorch_lightning.utilities.distributed import rank_zero_info
+
+from lasaft.source_separation.conditioned.scripts import trainer as trainer
+from lasaft.utils.functions import mkdir_if_not_exists
+
+
+def main(cfg: DictConfig):
+    # Load config
+    rank_zero_info(OmegaConf.to_yaml(cfg))
```

Comment:
What is it for btw? I searched it but couldn't find relevant articles.

Reply:
Ah, it implemented in pytorch lightning (https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/distributed.py).

Use to enable logging only at the prompt in rank 0, not at all prompts in a distributed environment.